Makefile

```diff
@@ -37,8 +37,13 @@ pre-run:
 	@rm -rf selectors
 	@rm -rf store
 	@rm -rf utils
+	@rm -rf lib
 
 test: check-style
 	npm test
 
 install: .npminstall
+
+bundle:
```

Comment:
Doesn't this require the install target?

Reply:
I don't think it does. It works, ie `make bundle` does what I expect it would...